apps/web-giddh/src/app/ledger/ledger.component.ts

```
@@ -2177,9 +2182,9 @@ export class LedgerComponent implements OnInit, OnDestroy {
             entryUniqueNames = [...debitCreditTransactions?.filter(debitCreditTransaction => debitCreditTransaction.isChecked).map(debitCreditTransaction => debitCreditTransaction.entryUniqueName)];
         } else {
             entryUniqueNames = [
-                    ...debitTrx?.filter(f => f.isChecked).map(dt => dt.entryUniqueName),
-                    ...creditTrx?.filter(f => f.isChecked).map(ct => ct.entryUniqueName),
-                ];
+                ...debitTrx?.filter(f => f.isChecked).map(dt => dt.entryUniqueName),
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)